api/opencl_runtime_layer.asciidoc

```diff
@@ -14781,6 +14788,7 @@ Executable:: State after command recording has finished with
 // stateDiagram-v2
 //     [*] --> Recording: Create
 //     Recording -->Executable: Finalize
+//     Executable
```

Comment:
This comment doesn't match the new PNG lifecyle diagram you've created, e.g. there is no "Finalized" state.